girder/api/v1/user.py

```diff
@@ -235,11 +235,11 @@ def updateUser(self, user, firstName, lastName, email, admin, status):
         user['email'] = email
 
         # Only admins can change admin state
-        if admin is True:
+        if admin in {True, False}:
```

Comment:
Doesn't `autoDescribeRoute` already coerce `admin` to a boolean, when provided? Can this be `if admin is not None:`?